pkg/controller/inference/service_controller.go

```diff
@@ -159,7 +165,7 @@ func (r *ServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func buildWorkloadApplyConfiguration(service *inferenceapi.Service, models []*coreapi.OpenModel) *applyconfigurationv1.LeaderWorkerSetApplyConfiguration {
+func buildWorkloadApplyConfiguration(service *inferenceapi.Service, models []*coreapi.OpenModel, initContainerImage string) *applyconfigurationv1.LeaderWorkerSetApplyConfiguration {
```

Comment:
Maybe we should pass the configuration, we may have more config fields in the future.

Reply:
Agree with that.